setup.py

```diff
@@ -23,6 +22,7 @@
         'Programming Language :: Python :: 3.11',
         'Programming Language :: Python :: 3.12',
     ],
+    python_requires='>=3.5',
```

Comment:
is there any reason why Python versions that are no longer receiving (security) updates should still be supported?

Python 3.8 will be EOL in October 2024, so I would aim to support only Python 3.8+.

Reply:
I am just doing a slight cleanup, aligning the metadata with what the project claims. I could totally remove deprecated python version support in a different dedicated PR.

@sergei-maertens do you have the merge permission?

Reply:
I do not, I'm mostly tracking upstream changes to maintain our own fork

Reply:
I start to think the project is abandonned.